### DISCUSSION TITLE
Do you have any plans to support the Eclipse IDE icon?

### DISCUSSION THREAD

**Discussion by HwangBBang** [devicons/devicon#1214] [Ideas]
If there is, I think it will be very useful.

**Comment by amacado** [comment]
Feel free to create an issue for eclipse: https://github.com/devicons/devicon/issues/new?assignees=&labels=request%3Aicon&template=icon_request.yml&title=%5BICON+REQUEST%5D%3A+%3Cicon+name%3E
